### ISSUE TITLE
Project-wide statement on data, consent, and GDPR?

### ISSUE THREAD

**Issue by mbbroberg** [chaoss/governance#227]
I've been reviewing metrics and thought about this concern. Do we have a project-wide statement on data and consent? 

I'm thinking whether we consider people should "get started" with CHAOSS, we add a note about it with further reading. I don't believe it's our responsibility to inform everyone about GDPR and how to manage data, but I do think it's right to point out the best practice of only share data your community would feel comfortable seeing publicly. 

Wdyt?

**Comment by germonprez** [comment]
Thanks @mbbroberg! 

This was the result of our June 16th meeting. I'm guessing we should just add this to the webpage somewhere. 

We can make a disclaimer in our metrics release:

The CHAOSS project recognizes that there are ethical and legal challenges when using the metrics and software provided by the community. The specific challenges arise in the use that is specific to your context. 

Ethical challenges are around protecting our community members and empowering them with their personal information.
Legal challenges are around GDPR and similar laws that protect personal information of our community members. 

The CHAOSS project strongly recommends you consult a lawyer for legal challenges and work with your community to mitigate ethical challenges.

**Comment by GeorgLink** [comment]
The disclaimer is already on the metric page.

Maybe we do want to have more somewhere on the website?

**Comment by mbbroberg** [comment]
@GeorgLink @germonprez I like the idea of a "Regarding Data Collection" page that is basically what Matt wrote above. I'd then PR a link to it in the root README of each WG. How's that sound?

**Comment by germonprez** [comment]
Full disclosure, I didn't write that :) Maybe I wrote parts but this was collaboratively written in one of the community calls. 😄 

Having this on the WG Readme pages is a good idea. We may want to find a spot on /website too.

**Comment by mbbroberg** [comment]
Whatever we do, one copy would be best. If not the website, let's PR into this repo and link out. We don't want to update the text in X different orgs when we want to change it.

**Comment by GeorgLink** [comment]
Good idea. I would prefer a page on the website if we link to it as a central resource.

**Comment by Nebrethar** [comment]
This was brought back to our attention in the meeting today. [Minutes here](https://docs.google.com/document/d/1PMDWc6xMe0fNE7shxTK5_HE_ykRBG5w55_Zx5hvzsEY/edit).
